crates/parcel/src/plugins.rs

```diff
@@ -57,7 +56,7 @@ impl<'a> Plugins<'a> {
   }
 
   pub fn bundler(&self) -> Result<Box<dyn BundlerPlugin>, anyhow::Error> {
-    Ok(Box::new(NapiBundlerPlugin::new(
+    Ok(Box::new(PluginBundlerRpc::new(
```

Comment:
Naming should be `RpcBundlerPlugin` and for the rest too

Reply:
Updated